addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -287,7 +287,13 @@ registerModel({
             const HIDDEN_MENU_WIDTH = 200; // max width, including width of dropup list items
             const START_GAP_WIDTH = device.isMobile ? 0 : 10;
             const chatWindows = this.allOrdered;
+<<<<<<< HEAD
             if (!device.isMobile && discuss.discussView) {
+||||||| parent of 67a67d19d5c (temp)
+            if (!device.isMobile && discuss.isOpen) {
+=======
+            if ((!device.isMobile && discuss.isOpen) || this.messaging.discussPublicView) {
+>>>>>>> 67a67d19d5c (temp)
```

Comment:
`discuss.isOpen` was changed to `discuss.discussView`
Resolving conflict just consists of adding `|| this.messaging.discussPublicView` like before